HW_2/HW_2.Rmd

```diff
@@ -0,0 +1,129 @@
+---
+title: "HW_2"
+author: "Antonietta"
+date: "2/6/2021"
+output:   
+  github_document: default
+  html_document: default
+  keep_md: true
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+library(gapminder)
+library(tidyverse)
+library(palmerpenguins)
+library(gapminder)
+library(ggplot2)
+
+```
+
+
+# Exercise 1 : Basic dplyr
+
+## Task 1: filter () subset gapminder  data to 3 countries; France, Japan, and Italy and pipe %>% to select country and gdpPercap 
+
+```{r filter , echo=FALSE}
+gapminder %>%
+  select(country, year, gdpPercap) %>%
+  filter(country %in% c ("France" , "Japan", "Italy"), year>1969 & year <1980)
+
```

Comment:
I think you want to reverse these two functions. As of now you are limited the variables you subset from before you subset. It may not have a lot of influence in the long run, but I just wanted to point it out.

Reply:
This is my first time coding, what does reversing the functions mean? I'd definitely want to fix this long term!

Reply:
So from what I understand r will go through your code one step at a time. So here you are telling it to take the gapminder data, then selected out these variables, then subset from that set of variables into the specific ones you want here (those three countries between those years). You could first subset the data so it takes all the variables associated with France Japan and Italy between the years of 1970 to 1979 and then pulls out the specific variables you want (country and gdpPercap). This would just mean putting the select line after the filter line.